ipdb/__main__.py

```
@@ -49,10 +62,17 @@ def _get_debugger_cls():
     return shell.debugger_cls
 
 
-def _init_pdb(context=None, commands=[]):
+def _init_pdb(context=None, ipython_profile=None, commands=[]):
```

Ruff (0.15.2)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)